src/main/java/org/eclipse/tractusx/ssi/lib/model/JsonLdObject.java

```diff
@@ -51,7 +51,7 @@ public abstract class JsonLdObject extends HashMap<String, Object> {
    *
    * @param json the json
    */
-  public JsonLdObject(Map<String, Object> json) {
+  protected JsonLdObject(Map<String, Object> json) {
```

Comment:
Why is the visibility changed here?

Reply:
So far this object is accessed within package only, again this a sonar suggestion